pywsitest/ws_message.py

```diff
@@ -28,16 +29,16 @@ def __init__(self):
         self.attributes = {}
         self.delay = 0.0
 
-    def __str__(self):
+    def __str__(self) -> str:
         # Output the attributes dictionary as json
         return json.dumps(self.attributes)
 
-    def with_attribute(self, key, value):
+    def with_attribute(self, key: str, value: object) -> "WSMessage":
```

Comment:
that is a funny syntax, but I guess that's what we can do now (other than doing a __future__ import)

Reply:
I may end up using the new future import, as there's another python 3.7 feature I want to take advantage of in the async code.
Will leave it as is for now though